<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="db" uuid="72aae3cf-7625-40f2-8d67-4aca97fdd25a">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\10\PycharmProjects\internet-engineering-project\db.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="db [2]" uuid="2c287f94-178f-40d8-9db0-a436007aef95">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/db.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="bb9e68bd-b62b-4d6b-8b21-45f045167034">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/helli5/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/db.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="ed811377-48d4-4ad8-9361-9f97ff613bd5">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/helli5/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/db.sqlite3</jdbc-url>
    </data-source>
  </component>
</project>